src/layman/map/micka/csw.py

```diff
@@ -160,16 +160,14 @@ def get_template_path_and_values(publication: Map, *, http_method=None, actor_na
     ]))), None)
 
     prop_values = _get_property_values(
-        workspace=publication.workspace,
-        mapname=publication.name,
         uuid=publication.uuid,
         title=title,
         abstract=abstract or None,
         publication_date=publ_datetime.strftime('%Y-%m-%d'),
         revision_date=revision_date.strftime('%Y-%m-%d'),
         md_date_stamp=date.today().strftime('%Y-%m-%d'),
-        identifier=url_for('rest_workspace_map.get', workspace=publication.workspace, mapname=publication.name),
-        identifier_label=publication.name,
+        identifier=url_for('rest_map.get', uuid=publication.uuid),
+        identifier_label=f"m-{publication.uuid}",
```

Comment:
Metadata properthies distinguished two identifiers:
- `md_file_identifier` that is identifier of metadata record; this one has `m-` prefix
- `identifier` that is identifier of the publication; this one does not have `m-` prefix

Because this is label of `identifier` of the map, it should not have `m-` prefix. Please change it to

```suggestion
        identifier_label=publication.uuid,
```